README.md

```diff
@@ -116,6 +116,52 @@ X's indicate that these details have not been determined as of commit date.
 
 Approximately three months prior to a scheduled software upgrade, a branch from Master will be created with the new release version tag. Pull requests that address bugs should then be made to both Master and the new release branch. Pull requests that require extensive review and testing (generally, optimizations and new features) should *not* be made to the release branch. 
 
+<<<<<<< HEAD
```

Comment:
Why did this get approved? Has @fluffypony _ever_ rejected or commented on a PR? To me it seems they simply get merged without further look.

Reply:
@leonklingele what do you expect from a guy who thinks merging code is a button mashing exercise? [[ 1 ]](https://twitter.com/fluffypony/status/984507752243490817)